Permanent/Modules/Authentication/Screens/Register/RegisterView.swift

```diff
@@ -0,0 +1,220 @@
+                                .layoutPriority(1)
+                                .padding(.trailing, 2)
+                            }
+                            RoundButtonRightImageView(text: "Sign Up", rightImage: Image(.rightArrowShort), action: {
+                                
+                            })
+                            .opacity(!viewModel.agreeTermsAndConditions ? 0.5 : 1)
+                            .disabled(!viewModel.agreeTermsAndConditions)
+                            .id(passwordFieldId)
+                            
+                            
+                            HStack(spacing: 20) {
+                                Rectangle()
+                                    .foregroundColor(.clear)
+                                    .frame(height: 1)
+                                    .background(Color(red: 0.54, green: 0.55, blue: 0.64))
+                                    .layoutPriority(0.5)
+                                Text("Already registered?".uppercased())
+                                    .font(
+                                        .custom("Usual-Regular", size: 10))
+                                    .kerning(1.6)
+                                    .lineLimit(1)
+                                    .multilineTextAlignment(.center)
+                                    .foregroundColor(Color(red: 0.72, green: 0.73, blue: 0.79))
+                                    .layoutPriority(1)
+                                Rectangle()
+                                    .foregroundColor(.clear)
+                                    .frame(height: 1)
+                                    .background(Color(red: 0.54, green: 0.55, blue: 0.64))
+                                    .layoutPriority(0.5)
+                                
+                            }
+                            SmallRoundButtonImageView(type: .noColor, imagePlace: .onRight, text: "Sign in", image: Image(.authRegisterKey), action: {
+                                self.viewModel.containerViewModel.setContentType(.login)
+                            })
+                            .padding(.bottom, Constants.Design.isPhone ? 2 : 3)
+                            .id(bottomButtonId)
+                            
+                            if !showEmptySpace {
+                                Spacer()
+                                    .frame(height: keyboardHeight - 64 > 0 ? keyboardHeight - 64 : keyboardHeight)
+                            }
+                        }
+                    }
+                    .frame(height: geometry.size.height)
+                    .onAppear {
+                        UIScrollView.appearance().bounces = false
+                    }
+                    .onDisappear {
+                        UIScrollView.appearance().bounces = true
+                    }
+                }
+            }
+        }
+        .onChange(of: viewModel.loginStatus, perform: { status in
```

Comment:
Do you need this one?

Reply:
This will be used/modifed when the API call will be implemented.